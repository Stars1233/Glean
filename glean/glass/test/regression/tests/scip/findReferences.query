action: findReferenceRanges
args: "test/rs/rust-analyzer/cargo/relative_path/./%28self%29"
