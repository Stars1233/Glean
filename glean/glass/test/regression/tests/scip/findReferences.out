[
    "@generated",
    [
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 18,
                "columnEnd": 22,
                "lineBegin": 33,
                "lineEnd": 33
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 15,
                "columnEnd": 19,
                "lineBegin": 55,
                "lineEnd": 55
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 9,
                "columnEnd": 13,
                "lineBegin": 92,
                "lineEnd": 92
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 9,
                "columnEnd": 13,
                "lineBegin": 96,
                "lineEnd": 96
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 9,
                "columnEnd": 13,
                "lineBegin": 100,
                "lineEnd": 100
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 9,
                "columnEnd": 13,
                "lineBegin": 104,
                "lineEnd": 104
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 9,
                "columnEnd": 13,
                "lineBegin": 108,
                "lineEnd": 108
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 9,
                "columnEnd": 13,
                "lineBegin": 112,
                "lineEnd": 112
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 25,
                "columnEnd": 29,
                "lineBegin": 116,
                "lineEnd": 116
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 43,
                "columnEnd": 47,
                "lineBegin": 116,
                "lineEnd": 116
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 28,
                "columnEnd": 32,
                "lineBegin": 122,
                "lineEnd": 122
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 41,
                "columnEnd": 45,
                "lineBegin": 122,
                "lineEnd": 122
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 9,
                "columnEnd": 13,
                "lineBegin": 138,
                "lineEnd": 138
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 19,
                "columnEnd": 23,
                "lineBegin": 140,
                "lineEnd": 140
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 14,
                "columnEnd": 18,
                "lineBegin": 146,
                "lineEnd": 146
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 25,
                "columnEnd": 29,
                "lineBegin": 154,
                "lineEnd": 154
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 52,
                "columnEnd": 56,
                "lineBegin": 156,
                "lineEnd": 156
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 30,
                "columnEnd": 34,
                "lineBegin": 194,
                "lineEnd": 194
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 30,
                "columnEnd": 34,
                "lineBegin": 207,
                "lineEnd": 207
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 51,
                "columnEnd": 55,
                "lineBegin": 222,
                "lineEnd": 222
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 50,
                "columnEnd": 54,
                "lineBegin": 223,
                "lineEnd": 223
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 30,
                "columnEnd": 34,
                "lineBegin": 255,
                "lineEnd": 255
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 29,
                "columnEnd": 33,
                "lineBegin": 256,
                "lineEnd": 256
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 29,
                "columnEnd": 33,
                "lineBegin": 257,
                "lineEnd": 257
            },
            "repository": "test"
        },
        {
            "filepath": "relative_path.rs",
            "range": {
                "columnBegin": 31,
                "columnEnd": 35,
                "lineBegin": 258,
                "lineEnd": 258
            },
            "repository": "test"
        }
    ]
]